src/main/java/gov/cabinetofice/gapuserservice/dto/UserDto.java

```diff
@@ -15,6 +20,8 @@ public class UserDto {
     private List<Role> roles;
     private Role role;
     private Department department;
+    private Optional<Instant> created;
```

Comment:
We don't want to use Optionals as a field type, instead we can make them @Nullable

```
@Nullable
    private Instant created;
    @Nullable
    private Instant updated;

    public UserDto(final User user) {
        this.gapUserId = user.getGapUserId().toString();
        this.emailAddress = user.getEmailAddress();
        this.sub = user.getSub();
        this.department = user.getDepartment();
        this.roles = user.getRoles();
        this.role = user.getHighestRole();
        this.created = user.getCreated();
        this.updated = user.getUpdated();
    }
```